tracim/models/contents.py

```diff
@@ -251,6 +251,16 @@ def __init__(self, slug: str):
                 return
         raise ContentTypeNotExist()
 
+    def alias(self) -> typing.List[str]:
```

Comment:
this method should be named: 

- `get_aliases` if used like a function
- `aliases` or `alias_list if used like a property (in this case, use the `@property` decorator)

Reply:
by the way what is the concept behind an alias? is it a list of slugs? If so, why do you use a new term/concept? the right naming should be `slugs` or `slug_list`.

Reply:
This code is needed to know all "valid" slug related to one content_type, this is useful to deal with legacy naming trouble as "page"-> "html-documents" .

We can rename this slugs but i'm not sure that really clear, as there is officially "one correct slug", but we should know that "pages" should be mapped as "html-documents"

Reply:
I propose "get_slug_aliases"